src/book/03-guides/03-hardware/03-batteries.mdx

```diff
@@ -0,0 +1,86 @@
+---
+section: Guides
+chapter: Hardware
+title: Batteries
+description: How to safely work with batteries
+slug: /guides/hardware/batteries
+authors:
+  - Lachlan Court (@LachlanCourt)
+---
+
+## Battery Safety
+
+<Alert>This page is not designed to be a substitute for the official lab induction, however it should be suitable as a revision tool</Alert>
+
+Our robots have the capability to run on both battery power and by plugging into a power supply. The robots should be run on battery power for events such as
+RoboCup, RoboCup qualification videos, lab inductions, and other times were a cable attached to the robot may intefere with its operation. During standard testing in
+the lab, you will usually have the robot running off a power supply.
+
+Our NUgus robots run on 4 Cell 14.8V Lithium Polymer (LiPo) batteries, whilst the Darwin robots use a smaller 3 Cell 11.1V model. In order to handle batteries in the
+lab you should have completed a formal lab induction.
+
+LiPo batteries can be extremely volatile and should always be handled with the utmost care. When not being used, they should be kept in fire retardant bags and stored
+carefully in either the "Flat" or "Charged" drawer depending on its current state.
+After use, a battery should be be inspected for any physical damage, and clearly marked and set aside if any is present.
+
+## Using in the robot
+
+<Alert type="warning">Never use a battery in the robot without a battery monitor</Alert>
+
+<Alert type="warning">Always unplug both the robot and the monitor from the battery when not being used. Both of these can lead to the battery going flat beyond
+recovery</Alert>
+
+Whenever a battery is being used in the robot, it should be additionally connected to a battery monitor. This ensures that the battery will not run so low that it
+will be unusably flat. Unlike regular batteries, LiPo batteries cannot run completely flat unless they are due to be disposed of.
+
+To connect a battery monitor to the battery:
+
+1. Start by finding the cable on the battery with the white plug. Line up the `-` symbol on the monitor with the 5 black leads of the battery cable, and plug in. Be
+   aware that the battery monitor will beep several times when this happens, this is normal. ![Battery monitor attached to a battery](./images/batteries/battery_monitor.jpg)
+
+2. Once the battery monitor is attached to the battery, the other cable on the battery can now be plugged into the robot. Depending on the battery this may require an
+   adaptor lead, shown below.
+
+3. Once both cables on the battery are hooked up, the robot can safely be turned on. Be sure to carefully load the battery into the robot to avoid damage, this may
+   require some technique so don't be afraid to ask a team member for the best way to go about this.
```

Comment:
Split these up into 2 points
```suggestion
3. Once both cables on the battery cables are hooked up, install the battery into the robot and install the back panel cover. Be sure to carefully load the battery into the robot to avoid damage, this may require some technique so don't be afraid to ask a team member for the best way to go about this.
4. The robot can now be safely turned on.
```